.changelog/4003.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+api-gateway: fix bug where multiple logical APIGateways would share the same ACL policy.
+```
```

Comment:
Why do you need 2 changelogs?

Reply:
because I committed this one on the wrong branch :joy: